smac/intensification/successive_halving.py

```diff
@@ -351,23 +357,33 @@ def eval_challenger(self,
                                   "Interrupting current challenger and moving on to the next one")
                 # ignore all pending instances
                 self.curr_inst_idx = np.inf
+                status = StatusType.CAPPED
```

Comment:
Isn't this already set by the tae?

Reply:
Yes, but only in run history. When we get a `CappedRunException`, we don't get outputs from the `ExecuteTARun` in the try block. So we need to set it explicitly here.